src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx

```diff
@@ -96,54 +96,14 @@ Our dashboards take the data you put into New Relic and provide you with a wide
 
 ## Your alerting solution [#alerts-solutions]
 
-Being able to see your data is great, but being able to act on it to correct or prevent errors is even better! New Relic gives you multiple ways to customize alerts for your unique requirements while helping to eliminate alert fatigue. We offer proactive awareness for your problems, utilizing machine learning to enhance our policies. Our alerts:
+Being able to see your data is great, but being able to act on it to correct or prevent errors is even better! New Relic gives you multiple ways to customize <InlinePopover type="alerts" /> for your unique requirements while helping to eliminate alert fatigue. We offer proactive awareness for your problems, utilizing machine learning to enhance our policies. Our alerts:
```

Comment:
I think same here?

Reply:
It's just providing an easy link into the UI. We throw these in across the docs site. It's just a convenience for people who may not know where things are in the UI. This seems useful particularly as we're removing alerts from the install flow in the UI.

Reply:
Thanks for the clarification! :)